lite/libxmp-lite-config-version.cmake.autotools

```diff
@@ -0,0 +1,21 @@
+set(PACKAGE_VERSION "4.5.1")
```

Comment:
Are you sure this version number would be updated and not forgotten? If you making a template for Autotools, please make sure to automatise this case.